pom.xml

```diff
@@ -21,265 +27,56 @@
         <url>https://github.com/elucidation-project</url>
     </organization>
 
-    <developers>
-        <developer>
-            <name>Chris Rohr</name>
-            <url>https://gitlab.com/chrisrohr</url>
-        </developer>
-        <developer>
-            <name>Terez Ivy</name>
-            <organization>Fortitude Technologies, LLC</organization>
-            <organizationUrl>https://gitlab.com/fortitudetec</organizationUrl>
-            <url>https://gitlab.com/Terezivy</url>
-        </developer>
-        <developer>
-            <name>Scott Leberknight</name>
-            <organization>Fortitude Technologies, LLC</organization>
-            <organizationUrl>https://gitlab.com/fortitudetec</organizationUrl>
-            <url>https://gitlab.com/sleberknight</url>
-        </developer>
-    </developers>
-
     <contributors>
         <contributor>
             <name>John Loizeaux</name>
             <url>https://gitlab.com/loizeaux</url>
         </contributor>
     </contributors>
 
-    <licenses>
-        <license>
-            <name>MIT License</name>
-            <url>http://www.opensource.org/licenses/mit-license.php</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
     <scm>
         <connection>scm:git:https://github.com/elucidation-project/elucidation.git</connection>
         <developerConnection>scm:git:git@github.com:elucidation-project/elucidation.git</developerConnection>
         <url>https://github.com/elucidation-project/elucidation</url>
         <tag>HEAD</tag>
     </scm>
 
-    <distributionManagement>
-        <snapshotRepository>
-            <id>sonatype-nexus-snapshots</id>
-            <name>Sonatype Nexus Snapshots</name>
-            <url>${sonatypeOssDistMgmtSnapshotsUrl}</url>
-        </snapshotRepository>
-        <repository>
-            <id>sonatype-nexus-staging</id>
-            <name>Nexus Release Repository</name>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
-
     <modules>
-        <module>elucidation-bom</module>
```

Comment:
I missed during my initial review that this is going away entirely. Might it be better to keep elucidation BOM, but have it import kiwi-bom as the only thing in it right now? This would allow new dependencies to be added and controlled which are specific to elucidation. Thoughts?